addons/calendar/models/calendar_event.py

```diff
@@ -364,8 +380,10 @@ def create(self, vals_list):
                                 activity_vals['user_id'] = user_id
                             values['activity_ids'] = [(0, 0, activity_vals)]
 
+        # Automatically add the current partner when creating an event if there is none (happens when we quickcreate an event)
+        self_partner_id = [(4, self.env.user.partner_id.id)]
```

Comment:
There is already a default (not the clearest one I agree): _default_partners . Shouldn't we ensure default is always valid instead of hardcoding it here ?